GraphicEditor/View/UserControls/CardiologyFirstFloorMapUserControl.xaml

```diff
@@ -0,0 +1,450 @@
+                        Style="{StaticResource ResourceKey=DefaultBuildingButtonStyle}">
+                    <StackPanel>
+                        <TextBlock Style="{StaticResource MaterialDesignHeadline5TextBlock}" 
+                                   TextWrapping="Wrap">
+                                Room4
+                        </TextBlock>
+                    </StackPanel>
+                </Button>
+
+                <!--Room5 button-->
+                <Button x:Name="Room5Button"
+                        Background="#FF237793" 
+                        BorderBrush="Black"
+                        CommandParameter="room5"
+                        Height="Auto"
+                        Margin="602,484,439,126"
+                        Style="{StaticResource ResourceKey=DefaultBuildingButtonStyle}">
+                    <StackPanel>
+                        <TextBlock Style="{StaticResource MaterialDesignHeadline5TextBlock}" 
+                                   TextWrapping="Wrap">
+                                Room5
+                        </TextBlock>
+                    </StackPanel>
+                </Button>
+
+                <!--Toilet-->
+                <Border BorderBrush="Black"
+                        BorderThickness="3"
+                        Margin="344,484,621,126">
+                    <Rectangle 
+                           Fill="#FF6DAEC1">
+                    </Rectangle>
+                </Border>
+                <Image Source="pack://application:,,,/Resources/Icons/toilet.png"
+                       Height="60"
+                       Margin="376,495,657,145"/>
+
+                <!--Elevator-->
+                <Border BorderBrush="Black"
+                        BorderThickness="3"
+                        Margin="806,495,255,126">
+                    <TextBox x:Name="elevator" 
+                             Background="#FF2F3B46"
+                             Text="Elevator"
+                             TextAlignment="Center"
+                             FontSize="24" 
+                             FontWeight="Bold"/>
+                </Border>
+
+                <!--Waiting room-->
+                <Grid Background="#FFC1E2EA"
+                      DockPanel.Dock="Top"  
+                      Margin="376,306,255,257">
+                    <Border Background="Transparent" />
+                </Grid>
+                <Image Source="/GraphicEditor;component/Resources/Icons/chair.png"
+                       Margin="392,320,747,328"/>
+                <Image Source="/GraphicEditor;component/Resources/Icons/chair.png"
+                       Margin="482,320,657,328"
+                       RenderTransformOrigin="0.5,0.5">
+                    <Image.RenderTransform>
+                        <TransformGroup>
+                            <ScaleTransform ScaleY="1" 
+                                            ScaleX="-1"/>
+                            <SkewTransform AngleY="0"
```

Comment:
Don't use this if it doesn't change anything

Reply:
This part changes the position of an icon, it flips her.